build.gradle

```diff
@@ -0,0 +1,159 @@
+import java.text.SimpleDateFormat
+
+buildscript {
+    ext {
+        springBootVersion = '1.5.2.RELEASE'
+    }
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        jcenter()
+    }
+    dependencies {
+        classpath 'io.github.robwin:jgitflow-gradle-plugin:0.5.0'
+    }
+}
+
+plugins {
+    id 'java'
+    id 'findbugs'
+    id 'org.springframework.boot' version '1.5.9.RELEASE'
+    id 'jacoco'
+    id 'maven-publish'
+    id 'com.jfrog.artifactory' version '4.4.14'
+    id 'com.bmuschko.clover' version '2.2.0'
+}
+
+bootRepackage {
+    enabled = false
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    maven { url "https://airdev.jfrog.io/airdev/libs-releases/" }
+}
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+configurations {
+    // Exclude logback, we will use log4j
+    compile.exclude module: 'spring-boot-starter-logging'
+    // This is pulled in from ap-common, we don't want security
+    // on our endpoints, at least for now
+    compile.exclude module: 'spring-boot-starter-security'
+
+    // Avoid having to list the findbugs annotations twice
+    testCompile.extendsFrom compileOnly
+}
+
+dependencies {
+    compileOnly 'com.google.code.findbugs:annotations:3.0.1'
+    compile 'org.hibernate:hibernate-java8:5.0.12.Final'
+    compile 'org.opentestsystem.ap:ap-common:0.3.26'
+    compile 'org.springframework.boot:spring-boot-starter-log4j2'
+    compile 'org.springframework.boot:spring-boot-starter-data-jpa'
+    // Type mappings for hibernate -> PostgreSQL Json
+    compile 'com.vladmihalcea:hibernate-types-5:2.0.0'
+    clover 'org.openclover:clover:4.2.0'
+    testCompile 'org.springframework.boot:spring-boot-starter-test'
```

Comment:
I considered removing these test compile dependencies since the project at this time doesn't have tests.  I left them in because I figure there are more classes than just the entity classes that we'll share.